plugins/slack_webhooks/app/views/samson_slack_webhooks/_fields.html.erb

```diff
@@ -2,7 +2,7 @@
   <legend>Slack</legend>
 
   <% stage = form.object %>
-  <% stage.slack_webhooks.build unless stage.slack_webhooks.last.try(:webhook_url).blank? %>
+  <% stage.slack_webhooks.build unless stage.slack_webhooks.size > 0 && stage.slack_webhooks.last.webhook_url.blank? %>
```

Comment:
ahhhh it handles missing hook and empty hook differently ... how about:

`unless (last = stage.slack_webhooks.last) && last.webhook_url.blank?`